### PR TITLE
[PWD-2604] Custom Training Page - Program List

### PR DESCRIPTION
**Pull request by haikal1223**
### What type of PR is this? (_check all applicable_)

- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### CODE REVIEW

**Review by gangsararyop — APPROVED** [review]

**Review by kevintanes — APPROVED** [review]

File: src/views/corporate-training/hooks/useProgramDataFormatter.ts [hunk 1]
```diff
@@ -7,16 +7,19 @@ interface FormattedData extends AdditionalData {
...
   ctDescription: string
+  categoryTitle?: string
 }
 
 export const useProgramDataFormatter = ({
   programData,
   isJobConnector = false,
-  selectedMethod
+  selectedMethod,
+  multipleMethod = true
```
Comment (shadiqfauzi) [inline src/views/corporate-training/hooks/useProgramDataFormatter.ts:17]:
```suggestion
  multipleMethod = true
```
nama props multipleMethod disini rada kurang tepat gue rasa, karena tujuannya buat render [0] additionalData kan?

Comment (shadiqfauzi) [reply src/views/corporate-training/hooks/useProgramDataFormatter.ts:17]: `ignoreMutipleMethod` menurut gue lebih tepat, sambil dijelasin gunanya apa pake notes aja

Comment (haikal1223) [reply src/views/corporate-training/hooks/useProgramDataFormatter.ts:17]: hmmm bener mas, cuman gw tadi rada bingung aja namingnya wkwkwk. Ada suggestion ga mas? selectFirstAdditionalData oke gak ya?

Comment (haikal1223) [reply src/views/corporate-training/hooks/useProgramDataFormatter.ts:17]: ehh komen yang ignoreMultipleMethod tadi ga muncul di gw wkwk. Okehh boleh tuh mass

**Review by gangsararyop — APPROVED** [review]

**Pull request merged**